apitools/base/py/credentials_lib.py

```diff
@@ -352,7 +356,7 @@ def _ScopesFromMetadataServer(self, scopes):
     def GetServiceAccount(self, account):
         relative_url = 'instance/service-accounts'
         response = _GceMetadataRequest(relative_url)
-        response_lines = [line.rstrip('/\n\r')
+        response_lines = [six.ensure_text(line).rstrip('/\n\r')
```

Comment:
If we're going to remove the `unicode_literals` import, we should make sure we're stripping off unicode here (i.e. adding the `u` prefix: `rstrip(u'/\n\r')`)

Reply:
Makes sense to me! Made both changes. :)